packages/snaps-execution-environments/.browserslistrc

```diff
@@ -0,0 +1,3 @@
+# The minimum browser versions supported by MetaMask.
```

Comment:
Out of curiosity, have you tried bumping these higher to see if it impacts bundle size?

Reply:
Yes, doesn't seem to make a significant difference.